Fixes #13 Security Considerations (#45)
tls-attestation/use-cases-and-properties: draft-mihalcea-seat-use-cases.md

requirements for a solution that is agnostic to any specific attestation
technology (e.g., Trusted Platform Modules (TPMs), Intel TDX, AMD SEV, Arm CCA).

> ### Evidence Semantics

Reviewer: I am slightly confused by the fact that the title says “Evidence” but the body talks about Attestation Results.

Author: because of https://www.ietf.org/archive/id/draft-mihalcea-seat-use-cases-02.html#section-5-2.4.1

removed now.

This document does not define attestation appraisal
  policies. However, a Relying Party must be careful when interpreting
  Attestation Results. A "valid" attestation only means the Evidence is

>   authentic and correctly signed; it does not automatically mean the underlying

Reviewer: Can it be authentic without being correctly signed?

Author: was the same here: https://www.ietf.org/archive/id/draft-mihalcea-seat-use-cases-02.html#section-5-2.4.1

  system is "secure". The Relying Party must have a clear policy for what
  measurements, software versions, and security configurations are acceptable.

> 

Reviewer: I don’t understand the reasoning behind this new section. IMO, if anything in this space needs clarifying, a single reference to §8.5 of RFC9334 should be enough.

Author: It's not a new section: https://www.ietf.org/archive/id/draft-mihalcea-seat-use-cases-02.html#section-5-2.4.1

{{Section 10 of -rats-arch}} for more details about freshness in the context of
RA. This is formalized for attestation nonce in  {{ID-Crisis}}.

> Failure to bind attestation credentials tightly to the

Reviewer 2: ```suggestion
Failure to bind attestation evidence tightly to the
```

Author: I think credentials here means both evidence and attestation results.

  current connection would allow an adversary to replay or relay old or stolen, yet
  valid credentials from a compromised system, completely undermining the
  security goals.

## Negotiation and Capability Discovery

Peers have a secure mechanism to discover each other's support for RA, the
specific attestation formats they can produce or consume, and the attestation
models they support. This enables interoperability and allows for graceful
fallback for endpoints that do not support RA.

The negotiation of attestation capabilities must be secure.
  An active attacker must not be able to trick two parties that both support

>   attestation into negotiating a connection without it.

Reviewer 2: I propose to modify the last line as follows:
An active attacker must not be able to cause two attestation-capable peers to negotiate a connection in which attestation is not performed.

Author: Thanks. Used the proposed wording as is.

minimize this leakage, especially when a third-party Verifier is involved in the
protocol exchange.

### Verifier Trust and Privacy
In the Background Check model, the Relying Party communicates with the Verifier at the time of appraisal.
This reveals the Attester's identity and connection timing to the Verifier.
This also reveals to the Verifier that the Relying Party is communicating with the specific Attester.
If the Verifier is a third party, it can observe which Attesters are being appraised and when, potentially exposing client identity and other correlation information.

> Solutions should consider privacy-preserving attestation  to minimize the data revealed to the Verifier, as being discussed in the RATS working group.

Reviewer 2: ```suggestion
Solutions should consider privacy-preserving attestation techniques being developed in the RATS working group, to minimize the data revealed to the Verifier.
```

minimize this leakage, especially when a third-party Verifier is involved in the
protocol exchange.

### Verifier Trust and Privacy
In the Background Check model, the Relying Party communicates with the Verifier at the time of appraisal.
This reveals the Attester's identity and connection timing to the Verifier.
This also reveals to the Verifier that the Relying Party is communicating with the specific Attester.

> If the Verifier is a third party, it can observe which Attesters are being verified and when, potentially exposing client identity or business relationships.

Reviewer: ```suggestion
If the Verifier is a third party, it can observe which Attesters are being appraised and when, potentially exposing client identity and other correlation information.
```

minimize this leakage, especially when a third-party Verifier is involved in the
protocol exchange.

### Verifier Trust and Privacy

> In the Background Check model, the Relying Party

Reviewer 2: I propose to update the text as follows:
In the Background Check model, the Relying Party contacts the Verifier at attestation time. This reveals the Attester's identity and connection timing to the Verifier. If the Verifier is a third party, it can observe which
Attesters are being verified and when, potentially exposing client identity or business relationships. Solutions should consider privacy-preserving
attestation as being discussed in the RATS working group.

Author: Thanks, used your phrasing as a baseline and slightly modified it for clarity based on RATS. Please check.
